cmd/osquery-perf/agent.go

```diff
@@ -1626,8 +1630,8 @@ func main() {
 		munkiIssueProb              = flag.Float64("munki_issue_prob", 0.5, "Probability of a host having munki issues (note that ~50% of hosts have munki installed) [0, 1]")
 		munkiIssueCount             = flag.Int("munki_issue_count", 10, "Number of munki issues reported by hosts identified to have munki issues")
 		// E.g. when running with `-host_count=10`, you can set host count for each template the following way:
-		// `-os_templates=windows_11.tmpl:3,mac10.14.6.tmpl:4,ubuntu_22.04.tmpl:3`
```

Comment:
Is `mac10.14.6.tmpl` still used somewhere, or can it be deleted?

Reply:
it's still a valid template if someone wants to pass it into `--os_templates`, but i think we can delete it soon if it's unused.  Since it's an older OS we no longer support, i'm not sure there's much value.

Reply:
ok, feel free to merge